### DISCUSSION TITLE
When upload a PDF, it throws an error: "Something went wrong :/ cannot reshape array of size 28290 into shape (2387,1890,newaxis)"

### DISCUSSION THREAD

**Discussion by tonycky** [open-webui/open-webui#6086] [General]
# Bug Report

**Important: Before submitting a bug report, please check whether a similar issue or feature request has already been posted in the Issues or Discussions section. It's likely we're already tracking it. In case of uncertainty, initiate a discussion post first. This helps us all to efficiently focus on improving the project.**

**Let's collaborate respectfully. If you bring negativity, please understand our capacity to engage may be limited. If you're open to learning and communicating constructively, we're more than happy to assist you. Remember, Open WebUI is a volunteer-driven project maintained by a single maintainer, supported by our amazing contributors who also manage full-time jobs. We respect your time; please respect ours. If you have an issue, We highly encourage you to submit a pull request or to fork the project. We actively work to prevent contributor burnout to preserve the quality and continuity of Open WebUI.**

## Installation Method

Docker

## Environment

- **Open WebUI Version:**  v0.3.30
- **Operating System:** Ubuntu 20.04
- **Browser (if applicable):** Chrome

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

The pdf can be uploaded successfully

## Actual Behavior:

When upload a PDF, it throws an error: "Something went wrong :/ cannot reshape array of size 28290 into shape (2387,1890,newaxis)"

## Description

**Bug Summary:**
When upload a PDF, it throws an error: "Something went wrong :/ cannot reshape array of size 28290 into shape (2387,1890,newaxis)"

## Reproduction Details

**Steps to Reproduce:**
1. Go to Workspace
2. Click tab "Documents"
3. Add new document ( my pdf is https://www.immd.gov.hk/pdforms/ID(E)969.pdf)

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by ediiiz** [comment]
Are you using Standard Engine or Tika?

I was able to fix this by adding Tika inside docker-compose.yaml

I was getting this error if the documents (only?) contained images.

```yaml
tika:
    image: apache/tika:latest-full
    container_name: tika
    expose:
      - "9998"
    networks:
      - openwebui-net <- change to your network.
    restart: unless-stopped
```
